app/hotels/src/filter/stars/StarsCheckbox.js

```diff
@@ -0,0 +1,77 @@
+// @flow
+
+import * as React from 'react';
+import { View, Text, StyleSheet, TouchableWithoutFeedback } from 'react-native';
+import { MaterialIcons } from '@expo/vector-icons';
+import { Color } from '@kiwicom/react-native-app-common';
+
+type Props = {|
+  stars?: number,
+  hotels: number,
```

Comment:
Please try to write required props to the beginning so it's immediately obvious what is required.